styles.css

```diff
@@ -0,0 +1,8 @@
+#image {
+    max-width: 100%;
+      display: block;
+    margin: auto;
+    }    
+    #main {
+      text-align: center;
+    }
```

Comment:
I would recommend you to follow the linters so we can keep the same style through all the project :smile: 
(use two spaces instead of tab, and add a final line of code at the end of the file)